boa_engine/src/vm/call_frame.rs

```diff
@@ -89,7 +94,7 @@ pub(crate) struct TryStackEntry {
     pub(crate) num_loop_stack_entries: usize,
 }
 
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
I would also derive `Copy`, remove any potential use of `clone()`, and add some documentation to it. Also, if you could add some documentation to the `FinallyReturn` enumeration would also be great (even if not related).